src/common/PatchDB.cpp

```diff
@@ -1351,6 +1351,23 @@ int PatchDB::numberOfJobsOutstanding()
 
 std::string PatchDB::sqlWhereClauseFor(const std::unique_ptr<PatchDBQueryParser::Token> &t)
 {
+    auto protect = [](const std::string &s) -> std::string {
```

Comment:
I'm new to sqlite -- is this similar to [quote](https://www.sqlite.org/lang_corefunc.html#quote)?

Reply:
Yeah except I’m inbound from non sql so have to proctrft my c++ string

Reply:
cool :) thanks for explaining!